src/main/python/rlbot/agents/hivemind/subprocess_hivemind.py

```diff
@@ -0,0 +1,82 @@
+# from multiprocessing import Event
+# from signal import SIGTERM
+from subprocess import Popen
+
+import queue
+import os
+
+import rlbot
+from rlbot.botmanager.agent_metadata import AgentMetadata
+from rlbot.botmanager.bot_helper_process import BotHelperProcess
+from rlbot.utils.logging_utils import get_logger
+from rlbot.utils.structures.game_interface import get_dll_directory
+
+class SubprocessHivemind(BotHelperProcess):
```

Comment:
It looks like this class supports executable hive minds which prefer to have all indices passed to them as a command line argument, rather than launching with none and waiting for signals like ExecutableWithSocketAgent. That seems like a valid use case, and probably the thing you need to support the rust way of doing things right?

Reply:
Pretty much. I have not looked at languages that use sockets yet. Rust bots receive the index as a command line argument so I did it the same way here.